lib/sgtn-client/api/translation.rb

```diff
@@ -52,7 +60,12 @@ def self.getStrings(component, locale)
 
         default = SgtnClient::Config.configurations.default
         if items.nil? || items["messages"] == nil
-          items = SgtnClient::Source.getSources(component, default)
+          items = {}
+          s = SgtnClient::Source.getSources(component, default)
+          key, value = s.first
+          items["component"] = key
+          items["messages"] = value
+          items["locale"] = 'source'
```

Comment:
Don't suggest hardcode locale as 'source'.

Reply:
We can change it later(TODO) when the lib supports source locale configuration.